WORKSPACE

```diff
@@ -1,5 +1,10 @@
 workspace(name = "tf_serving")
 
+local_repository(
+    name = "org_tensorflow",
```

Comment:
Is it possible to keep this in workspace.bzl? If someone is using TF Serving as a submodule (not fully supported yet, but close), it would be ideal if they didn't need to include TF as well, and could just load workspace.bzl and call tf_serving_workspace().